x/bank/types/key.go

```diff
@@ -55,3 +55,17 @@ func AddressFromBalancesStore(key []byte) (sdk.AccAddress, error) {
 func CreateAccountBalancesPrefix(addr []byte) []byte {
 	return append(BalancesPrefix, address.MustLengthPrefix(addr)...)
 }
+<<<<<<< HEAD
+=======
```

Comment:
Merge conflict

Reply:
yes, I know, was writing about it in the Discord yesterday night. Thanks!